monitoring/rid_qualifier/host/config.py

```diff
@@ -10,22 +10,21 @@
 
 # These keys map to entries in the Config class
 KEY_AUTH_SPEC = 'AUTH_SPEC'
+ENV_KEY_REDIS_URL = os.environ['REDIS_URL']
```

Comment:
These values should be fixed and be located next to `ENV_KEY_PREFIX` above -- these are the names of the environment variables that used to populate each variable.  So, for instance, `ENV_KEY_REDIS_URL = 'REDIS_URL`